js/main.js

```diff
@@ -0,0 +1,28 @@
+var $imageUrl = document.querySelector('#image-url');
+var $imagePlaceholder = document.querySelector('.image-placeholder');
+
+function addPhoto(event) {
+  $imagePlaceholder.setAttribute('src', event.target.value);
+}
+$imageUrl.addEventListener('input', addPhoto);
+
+var $form = document.querySelector('form');
+
+function submitEntry(event) {
+  event.preventDefault();
+  var title = $form.elements.title.value;
+  var imageUrl = $form.elements.url.value;
+  var notes = $form.elements.notes.value;
+  var entryData = {
+    title,
+    imageUrl,
+    notes,
+    currentId: data.nextEntryId
```

Comment:
I would just name this property `id`. "current" doesn't really add anything, since this entry will always have the same id.